packages/pwa/app/components/forms/update-password-fields.jsx

```diff
@@ -22,7 +22,10 @@ const UpdatePasswordFields = ({form, prefix = ''}) => {
                 <Field {...fields.currentPassword} />
                 <Box>
                     <Button variant="link" size="sm" onClick={() => null}>
-                        <FormattedMessage defaultMessage="Forgot password?" />
+                        <FormattedMessage
+                            defaultMessage="Forgot password?"
+                            id="update_password_field.button.forgot_password"
```

Comment:
Same

Reply:
I've changed it to be a plural. Thanks for catching that